langchain4j/src/main/java/dev/langchain4j/memory/chat/NoopChatMemory.java

```diff
@@ -0,0 +1,27 @@
+package dev.langchain4j.memory.chat;
+
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.memory.ChatMemory;
+import java.util.List;
+
+/**
+ * An implementation of {@link ChatMemory} that does nothing.
+ * This is useful for simplifying the AiService code.
+ */
+public class NoopChatMemory implements ChatMemory {
```

Comment:
Where is this used? What's the difference between using this and no memory at all? Also this could probably be implemented as a window chat memory with size 0 (just noticing this, not necessarily saying that it is a good idea though).